pandas/core/resample.py

```diff
@@ -856,10 +856,32 @@ def size(self):
         # It's a special case as higher level does return
         # a copy of 0-len objects. GH14962
```

Comment:
this comment is no longer relevant I think (as this issue is closed).

Reply:
I have to say I'm not really sure what this comment is supposed to mean :)

Reply:
I think OK to remove

Reply:
OK, done.